CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for crate
 Unreleased
 ==========
 
+- Permit ``urllib3.Timeout`` instances for defining timeout values.
+  Using that, you can configure both the socket ``connect`` and ``read``
```

Comment:
Maybe reads better:
```
This way, both ``connect`` and ``read`` socket timeout settings can be configured, in seconds.
```

Reply:
Thanks for your suggestion. I used it 1:1, yet separated the mention of the unit into a separate sentence.

> Permit ``urllib3.Timeout`` instances for defining timeout values.
> This way, both ``connect`` and ``read`` socket timeout settings can be
> configured. The unit is seconds.